spec/requests/providers/proceeding_merits_task/attempts_to_settle_controller_spec.rb

```diff
@@ -101,7 +101,7 @@
 
         it "updates the task list" do
           subject
-          expect(legal_aid_application.legal_framework_merits_task_list.serialized_data).to match(/name: :attempts_to_settle\n\s+dependencies: \*\d\n\s+state: :complete/)
+          expect(legal_aid_application.legal_framework_merits_task_list.serialized_data).to match(/name: :attempts_to_settle\n\s+dependencies: \*\d\d\n\s+state: :complete/)
```

Comment:
How do you feel about switching to `\d+`?  `\d\d` could be fragile if the sequences change in the future.
It's only recently become an issue as we started expanding beyond the original list of 8 questions, `\d+` matches on one or more digits?
```suggestion
          expect(legal_aid_application.legal_framework_merits_task_list.serialized_data).to match(/name: :attempts_to_settle\n\s+dependencies: \*\d+\n\s+state: :complete/)
```

Reply:
Yeah that's perfect, thanks. 

I was just being lazy when making it go green.

It took me long enough to figure out why it was failing 😆

Reply:
Me too, the first time I hit it! 🤦‍♂️